serverlessworkflow/antora.yml

```
@@ -110,7 +110,7 @@ asciidoc:
     images_distributions_url: https://kie.apache.org/docs/start/download#container-images
     apple_support_url: https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unknown-developer-mh40616/mac
     kogito_examples_repository_url: https://github.com/apache/incubator-kie-kogito-examples
-    kogito_operator_repository_rawcontent_url: https://raw.githubusercontent.com/apache/incubator-kie-tools/refs/heads/{operator_version}/packages/sonataflow-operator
+    kogito_operator_repository_rawcontent_url: https://raw.githubusercontent.com/apache/incubator-kie-tools/refs/tags/{kogito_version}/packages/sonataflow-operator
```

isn't the same without "refs/tags"? To be more readable/simple:
```suggestion
    kogito_operator_repository_rawcontent_url: https://raw.githubusercontent.com/apache/incubator-kie-tools/{kogito_version}/packages/sonataflow-operator
```

---

this does not work thought - https://raw.githubusercontent.com/apache/incubator-kie-tools/10.0.0/packages/sonataflow-operator

---

we require the specific YAML file with `10.0.0` versioning and links

---

> this does not work thought - https://raw.githubusercontent.com/apache/incubator-kie-tools/10.0.0/packages/sonataflow-operator

but that's because with `raw.githubusercontent.com` you cannot access to a folder, just a file as "Makefile" inside that folder